docker/Dockerfile.cross

```diff
@@ -0,0 +1,22 @@
+# we build our images in 2 steps, to ensure we have a compact
+# image, since we want to add our current subdirectory
```

Comment:
This seems like it's a header, but it's more a comment on internals, needs a proper header I think

Reply:
Sounds good